source/includes/_endpoint-dataset.md

```diff
@@ -798,8 +799,14 @@ To pick which variable field to use on the `label` field on the SPSS variables,
 
 ###### CSV
 
-Categorical variable values will be exported as the category name by default.
-To use the category ids as data values, include `"use_category_ids": true` in the `"options"` attribute of the POST body.
+By default, categorical variable values will be exported using the category name 
+and missing values will use their corresponding reason string for all variables.
+
+The missing values will be exported with their configured missing reason in
+the CSV file. If specified on the `missing_values` export option, then all
+missing values on all columns will use such string instead of the reason.
+
+Refer to the options described on the table above for the `csv` format to change this behavior.
 
```

Comment:
Is this from an outdated version? The current content of the text describes, whatever string you send will be used.